src/PostgraphileNestedMutationsPlugin.js

```diff
@@ -641,32 +651,41 @@ module.exports = function PostGraphileNestedMutationPlugin(builder) {
               );
             }
 
-            await Promise.all(Object.keys(fieldValue).map(async k => {
-              if (k.includes('updateBy')) {
-                await Promise.all(
-                  (Array.isArray(fieldValue[k]) ? fieldValue[k] : [fieldValue[k]]).map(async (rowData) => {
-                    const resolver = pgNestedUpdateResolvers[foreignTable.id];
+            if (fieldValue.updateById || fieldValue.updateByNodeId) {
```

Comment:
This won't handle `updateBy[UniqueField]` cases

Reply:
Does this test case cover the case you describe?

```javascript
test(
  'forward nested mutation with nested updateByEmail',
  withSchema({
    setup: `
      create table p.parent (
        email text not null primary key,
        name text not null
      );
      
      create table p.child (
        id2 serial primary key,
        parent_email text not null,
        description text not null,
        constraint child_parent_fkey foreign key (parent_email)
          references p.parent (email)
      );

      insert into p.parent values('test@test.com', 'test parent');
      insert into p.child values(1, 'test@test.com', 'test child');
    `,
    test: async ({ schema, pgClient }) => {
      const query = `
        mutation {
          updateParentByEmail(
            input: {
              email: "test@test.com"
              parentPatch: {
                childrenUsingEmail: {
                  updateById2: {
                    id2: 1
                    childPatch: {
                      description: "renamed child"
                    }
                  }
                }
              }
            }
          ) {
            parent {
              email
              name
              childrenByParentEmail {
                nodes {
                  id2
                  parentEmail
                  description
                }
              }
            }
          }
        }
      `;
      expect(schema).toMatchSnapshot();

      const result = await graphql(schema, query, null, { pgClient });
      expect(result).not.toHaveProperty('errors');

      const data = result.data.updateParentByEmail.parent;
      expect(data.childrenByParentEmail.nodes).toHaveLength(1);
      data.childrenByParentEmail.nodes.map((n) =>
        expect(n.parentEmail).toBe(data.email),
      );
      expect(data.childrenByParentEmail.nodes[0].description).toEqual(
        'renamed child',
      );
    },
  }),
);
```

Reply:
Yes, completely. Also when using postgraphile with the --classicIds option enabled, `updateByNodeId` becomes `updateById` and `updateById` becomes `updateByRowId` (which can be quite confusing, but that's how relay works), that's why I was sticking with the more generic `str.includes('updateBy')`, but I think there is a nicer way of checking this, maybe using `pgNestedTableUpdaterFields`.

Reply:
That test is passing on master, but maybe there is a cleaner/better way to do the check.

Reply:
I misunderstood how it all worked on this specific point, thanks for taking the time to check it out and clear up my doubts! I am closing this discussion as everything is working as expected.